datalad_crawler/pipelines/tests/test_openfmri.py

```diff
@@ -80,7 +81,13 @@ def check_dropall_get(repo):
     clean(dataset=repo.path)  # remove possible extracted archives
     with assert_raises(AssertionError):
         ok_file_has_content(t1w_fpath, "mighty load 2.0.0")
-    repo.get('.')
+    try:
+        repo.get('.')
+    except CommandError as exc:
```

Comment:
Sorry, the `exc` is unused and leftover from debugging locally.  I can drop that, if you prefer.

Reply:
nah, that is ok.  
FWIW: openfmri pipeline is no longer actively used, since openfmri became openneuro which uses datalad natively.  But I kept pipeline as it might be excercising code which is otherwise would be untested